### PR TITLE
Fix MAP E2Es

### LINKED ISSUE #4751: Thai World Service Article Broken Locally Only
**Describe the bug**
NB: Live page looks fine: https://www.bbc.com/thai/international-49514085.amp

Issue identified debugging a cypress test, the following url http://localhost.bbc.com:7080/thai/international-49514085.amp shows following 'broken looking page':

image

The cypress test reports it cannot resolve this garbled image url: https://ichef.bbci.co.uk/news/800/undefined/ichef.bbci.co.uk/images/ic/$recipe/p07lyyc0.jpg 

If you look in the network tab you see the failed request:
image

A full local run of `npm run test:e2e` fails currently due this issue.

**To Reproduce**

In Browser: 
1. Go to http://localhost.bbc.com:7080/thai/international-49514085.amp 
1. See 'broken page' as above
1. Look in network tab to see various failed requests

Cypress Test
1. In terminal `export CYPRESS_ONLY_SERVICE=thai`, then `npm run test:e2e`
See failed cypress test:
image

**Expected behavior**
Not quite sure what is expected, sorry, someone please replace this with your expectation!

**Desktop (please complete the following information):**

- OS: iOS
- Browser: firefox
- Version 70

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by ryanmccombe**
Resolves https://github.com/bbc/simorgh/issues/4751

**Overall change:** _Resolves issues with MAP E2E tests._

**Code changes:**

- _MAPs were using a live asset in fixtures - this is problematic when running local tests, as the local embed endpoint uses test data._
- _Resolve a broken E2E test caused by special characters within CPS fixtures._

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

File: cypress/support/config/services.js [hunk 1]
```diff
@@ -1194,10 +1194,7 @@ const genServices = appEnv => ({
       },
       liveRadio: { path: undefined, smoke: false },
       mediaAssetPage: {
-        path:
-          isLive(appEnv) || isTest(appEnv)
-            ? undefined
-            : '/thai/international-49514085',
+        path: isLive(appEnv) ? undefined : '/thai/23122810',
```
Comment (sareh) [inline cypress/support/config/services.js]: This page currently 404s on Test `https://www.test.bbc.com/thai/23122810`
```suggestion
        path: isLive(appEnv) || isTest(appEnv) ? undefined : '/thai/23122810',
```

File: src/app/containers/CpsAssetPageMain/index.test.jsx [hunk 2]
```diff
@@ -84,13 +85,21 @@ describe('CpsAssetPageMain', () => {
     );
   });
 
-  it('should match snapshot for MAP', async () => {
+  it('should render MAPs', async () => {
```
Comment (sareh) [inline src/app/containers/CpsAssetPageMain/index.test.jsx]:
```suggestion
  it('should render MAP with paragraph text', async () => {
```

File: cypress/support/config/services.js (hunk 1 above)
Comment (ryanmccombe) [reply cypress/support/config/services.js]: Oops good catch - mozart not routing Thai MAPs yet

File: src/app/containers/ChartbeatAnalytics/index.jsx [hunk 3]
```diff
@@ -41,6 +41,7 @@ const ChartbeatAnalytics = ({ data }) => {
     service,
     sendCanonicalChartbeatBeacon,
     isCanonicalAndEnabled,
+    chartbeatConfig,
```
Comment (jamesdonoh) [inline src/app/containers/ChartbeatAnalytics/index.jsx]: This change is unexpected?

Comment (ryanmccombe) [reply src/app/containers/ChartbeatAnalytics/index.jsx]: Was a prettier autofix - does seem legitimate as it fixes an eslint warning - `34:6  warning  React Hook useEffect has a missing dependency: 'chartbeatConfig'. Either include it or remove the dependency array  react-hooks/exhaustive-deps` - can remove it though as not in scope for this PR

Comment (jamesdonoh) [reply src/app/containers/ChartbeatAnalytics/index.jsx]: OK makes sense now though I'd still leave it to be fixed separately!

**Review by jamesdonoh — APPROVED** [review]

File: src/app/containers/ChartbeatAnalytics/index.jsx (hunk 3 above)
Comment (ryanmccombe) [reply src/app/containers/ChartbeatAnalytics/index.jsx]: 👍 removed

**Review by sareh — APPROVED** [review]
The changes look good, in terms of fixing the end to end tests. 

In general, though, I'm not sure about having so much logic in the Cypress tests. We should separately decide whether we can be happy with more unit test coverage and only check paragraphs for example in that there's a non-empty paragraph element on the page.
